src/vendor/Path.ts

```diff
@@ -38,15 +39,30 @@ export default class Path {
   }
 
   /// normalizes the path
+  /// changes slashes for Windows
   /// throws if not an absolute path
   constructor(input: string | Path) {
     if (input instanceof Path) {
       this.string = input.string
-    } else if (!input || input[0] != '/') {
-      throw new Error(`invalid absolute path: ${input}`)
-    } else {
-      this.string = sys.normalize(input)
+      return
     }
+    switch (Deno.build.os) {
+      case "linux":
+      case "darwin":
+        if (!input || input[0] != '/') {
+          throw new Error(`invalid absolute path: ${input}`)
+        } else {	    
+          this.string = sys.normalize(input)	    
+        }
+        break
+      case "windows":               
+       if (!input || !(/^(["[a-zA-Z]:[\\\/].*|[\\\/].*)$/.test(input))) {          
```

Comment:
Seems like this is not a valid regex for the domain (`.*` rather than `.+`). However it seems enough to simply check for an initial `/^[a-zA-Z]:\\/` to me.

Reply:
I tried your suggestions and they didn't work unfortunately. We can have 0 following characters and we can also have paths like / or \\. So I left my initial regex.